pom.xml

```diff
@@ -24,7 +24,7 @@
     <profile>
       <id>ftest</id>
```

Comment:
Remove the `ftest` profile and just put `<module>ftest/startup</module>` in the default list of modules (to make @jcarsique happy 😬 ).